.eslintrc.cjs

```diff
@@ -46,6 +46,7 @@ module.exports = {
     "@typescript-eslint/no-unused-vars": [
       "warn",
       {
+        ignoreRestSiblings: true,
```

Comment:
This is accommodate filtering out values via destructuring like [below](https://github.com/la-ldaf/ldaf-site/pull/403/files#diff-948f3271a51b6b2180724f66411a30790e43053cd5f1236d8f9f53b821b1ed55R24)